### DISCUSSION TITLE
Help to Connect with My CRM

### DISCUSSION THREAD

**Discussion by lucaspmnt** [langflow-ai/langflow#2631] [General]
Hello everyone, I'm new to Langflow and don't know how to code.

I created a simple flow using RAG for a chatbot connected via API with n8n to support my company.

I would like help making a change by creating a tool (or agent?) so that when the user asks the chatbot for information such as registration status, payments, etc., the AI can retrieve this information from the CRM and respond to the user.

@dosu

![Screen Shot 2024-07-11 at 03 49 19](https://github.com/langflow-ai/langflow/assets/174758859/4a89cb6f-ca60-4303-a0c7-b1bf446ac640)

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
